### ISSUE TITLE
INTERNAL_SERVER_ERROR, Error: Node is not defined

### ISSUE THREAD

**Issue by Sharcoux** [d-kimuson/ts-type-expand#338]
The extension doesn't seem to work on my side. Here is the error I get:

```json
{
  "context": "ts-type-expand",
  "level": "error",
  "message": {
    "error": {
      "data": {
        "code": "INTERNAL_SERVER_ERROR",
        "httpStatus": 500,
        "path": "getTypeFromPos",
        "stack": "TRPCError: error: Error: Node is not defined\n    at /home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:4962\n    at HC.middlewares (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:618:122618)\n    at d (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:179)\n    at next (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:282)\n    at n (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:618:121620)\n    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async d (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:173)\n    at async next (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:276)\n    at async d (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:173)\n    at async next (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:276)"
      },
      "meta": {
        "response": {
          "size": 0
        }
      },
      "name": "TRPCClientError",
      "shape": {
        "code": -32603,
        "data": {
          "code": "INTERNAL_SERVER_ERROR",
          "httpStatus": 500,
          "path": "getTypeFromPos",
          "stack": "TRPCError: error: Error: Node is not defined\n    at /home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:4962\n    at HC.middlewares (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:618:122618)\n    at d (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:179)\n    at next (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:282)\n    at n (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:618:121620)\n    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async d (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:173)\n    at async next (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:276)\n    at async d (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:173)\n    at async next (/home/victus/.vscode/extensions/kimuson.ts-type-expand-1.0.1/node_modules/ts-type-expand-plugin/dist/index.js:627:276)"
        },
        "message": "error: Error: Node is not defined"
      }
    },
    "kind": "TRPC_SERVER_ERROR"
  }
}
```

**Comment by Sharcoux** [comment]
Maybe related to https://github.com/facebookarchive/draft-js/issues/522

**Comment by d-kimuson** [comment]
@Sharcoux 
Thanks for the report!
Are you using ts-type-expand on Widows machines? If so, unfortunately it is not currently supported to work on Windows machines(#327). I hope you will consider using WSL as a temporary workaround.

**Comment by Sharcoux** [comment]
Hi,
No, I'm on Ubuntu 22.04

**Comment by d-kimuson** [comment]
Oh, it is Ubuntu, OK.
I'll try to reproduce it on my ubuntu machine, but I'm busy and the fix may be a little while away.
Please wait.

**Comment by d-kimuson** [comment]
@Sharcoux 
Thank you for your patience!
The problem has been resolved in the v1.0.5 we just released, so we are closing it.
If it continues to occur, please comment on duplicate #353.

https://github.com/d-kimuson/ts-type-expand/releases/tag/1.0.5
